src/main/java/com/conveyal/r5/analyst/cluster/TravelTimeSurfaceRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.conveyal.r5.analyst.cluster;
+
+import com.conveyal.r5.profile.ProfileRequest;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Request a travel time surface containing travel times to all destinations for all iterations.
```

Comment:
Javadoc is no longer correct - this does not return travel times for all iterations, it returns a fixed number of percentiles.